L73-java-binderchannel/wireformat.md

```diff
@@ -201,7 +206,24 @@ ungraceful shutdown will occur, with the status code CANCELLED. The remote
 transport will be notified, cancelling all ongoing remote calls on the server
 side.
 
+## Large Messages
+
+We limit the amount of message data sent in a single transaction to 16KB.
+Messages larger than that will be sent in multiple sequential transactions, with
+all but the last transaction having FLAG_MESSAGE_DATA_IS_PARTIAL set.
+
+## Flow Control
+
+Due to the limited binder buffer size of 1MB, BinderTransport supports
```

Comment:
Consider: "Due to the limited binder buffer size of 1MB" -> "Due to Android's limited per-process transaction buffer of 1MB"

Adds a bit more context and makes it clear this buffer size is out of our control.

Reply:
Done. Thanks for the review and suggestions!